products/debian12/transforms/constants.xslt

```diff
@@ -0,0 +1,13 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:include href="../../../shared/transforms/shared_constants.xslt"/>
+
+<xsl:variable name="product_long_name">Debian 12</xsl:variable>
+<xsl:variable name="product_short_name">Debian 12</xsl:variable>
+<xsl:variable name="product_stig_id_name">DEBIAN_12_STIG</xsl:variable>
+<xsl:variable name="prod_type">debian12</xsl:variable>
+
+<!-- Define URI of official Center for Internet Security Benchmark for Debian Linux v1.0 -->
```

Comment:
The Debian 12 benchmark from CIS was not been released yet please remove this.

Reply:
I removed the URI and the STIG entry.